core/src/main/scala/com/salesforce/op/evaluators/OpBinScoreEvaluator.scala

```diff
@@ -134,20 +143,30 @@ private[op] class OpBinScoreEvaluator
  * Metrics of BinaryClassificationBinMetrics
  *
  * @param BrierScore            brier score for overall dataset
+ * @param binSize               size of each bin
  * @param binCenters            center of each bin
  * @param numberOfDataPoints    total number of data points in each bin
+ * @param sumOfLabels           sum of the label in each bin
```

Comment:
or do you mean count? - then I request a name change :-)

Reply:
It's meant to be the count of the "yes" labels, calculated as the sum of the labels. Does that make sense?

Reply:
that makes sense for the name but then see my comment below. Why the sum of yes and not just the count of all (that combined with the average gives you the same info but in a more intuitive formate)

Reply:
Isn't `numberOfDataPoints` the count of all?

Reply:
ah - good point :-) is what you want really the sum of the labels or the count of the positive labels? I dont think it is possible with the current models but say someone ran binary classification and fed in -1 and 1 instead of 0, 1 what do you want to see?

Reply:
Yes, what I'm wanting is the count of the positive labels. This could be reconstructed by multiplying the `averageConversionRate` with the `numberOfDataPoints` but figured it would be cleaner and less subject to round off error by having a parameter for it explicitly.

I assumed your -1/1 example wasn't possible but I can refactor the code to make sure it's only counting positive labels rather than just summing. Would `numberOfPositiveLabels` be an appropriate param name?

Reply:
Okay, fix is in! Thanks for the review!